### PR TITLE
ARGO-1176 API CALL - Get Service(s)

### PR DESCRIPTION
**Pull request by agelostsal**
Functionality ready.
Testing ready.
Docs and swagger updated

### CODE REVIEW

File: docs/swagger.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ swagger: '2.0'
 info:
   title: argo-api-authN
   description: "The purpose of the Authentication Service is to provide the ability to different services to use alternative authentication mechanisms without having to store additional user info or implement new functionalities.The AUTH service holds various information about a service’s users, hosts, API urls, etc, and leverages them to provide its function."
-  version: ""
+  version: "v1"
```
Comment (kaggis) [inline docs/swagger.yml]: versioning prefix needs to be added also as a base_path in swagger

File: docs/swagger.yml [hunk 2]
```diff
@@ -138,13 +182,22 @@ definitions:
       auth_method:
         type: string
         description: the authentication method that the sercice supports
-      retrieval_field:
+      retrieval_fiel:
```
Comment (kaggis) [inline docs/swagger.yml]: typo in json field

File: docs/swagger.yml [hunk 3]
```diff
@@ -138,13 +182,22 @@ definitions:
       auth_method:
         type: string
         description: the authentication method that the sercice supports
```
Comment (kaggis) [inline docs/swagger.yml]: small typo

File: docs/swagger.yml (hunk 1 above)
Comment (agelostsal) [reply docs/swagger.yml]: added

File: docs/swagger.yml (hunk 2 above)
Comment (agelostsal) [reply docs/swagger.yml]: fixed

File: docs/swagger.yml (hunk 3 above)
Comment (agelostsal) [reply docs/swagger.yml]: fixed

**Pull request merged**